src/dyn/Surface_Hopping.cpp

```diff
@@ -25,8 +25,15 @@ namespace libdyn{
 
 
 
-MATRIX compute_hopping_probabilities_fssh(CMATRIX& Coeff, CMATRIX& Hvib, double dt){
-/**
+MATRIX compute_hopping_probabilities_fssh(CMATRIX& Coeff, CMATRIX& Hvib, double dt, int use_boltz_factor, double T){

```

Comment:
Can't accept this change - it may break the compatibility with older developments. Please, just reload this function. In fact, you can wrap this one into the old-style signature with the additional parameters set up to the appropriate values (make sure it works as the older version)